test/collections.jl

```diff
@@ -58,13 +58,18 @@
 
     if visualtests
       Random.seed!(2021)
-      @test_reference "data/pset-1D-$T.png" plot(PointSet(rand(P1,10)))
-      @test_reference "data/pset-2D-$T.png" plot(PointSet(rand(P2,10)))
-      @test_reference "data/pset-3D-$T.png" plot(PointSet(rand(P3,10)))
-      @test_reference "data/pset-1D-$T-data.png" plot(PointSet(rand(P1,10)),1:10)
-      @test_reference "data/pset-2D-$T-data.png" plot(PointSet(rand(P2,10)),1:10)
-      @test_reference "data/pset-3D-$T-data.png" plot(PointSet(rand(P3,10)),1:10)
-    end
+      if VERSION < v"1.7.0"
+        ver = ""
+      else
+        ver = "_1.7"
+      end
+      @test_reference "data/pset-1D-$T$ver.png" plot(PointSet(rand(P1,10)))
+      @test_reference "data/pset-2D-$T$ver.png" plot(PointSet(rand(P2,10)))
+      @test_reference "data/pset-3D-$T$ver.png" plot(PointSet(rand(P3,10)))
+      @test_reference "data/pset-1D-$T-data$ver.png" plot(PointSet(rand(P1,10)),1:10)
+      @test_reference "data/pset-2D-$T-data$ver.png" plot(PointSet(rand(P2,10)),1:10)
+      @test_reference "data/pset-3D-$T-data$ver.png" plot(PointSet(rand(P3,10)),1:10)
```

Comment:
I think what we really want is to create a set of points deterministically and update the tests. We used random points because we wrote these tests quickly. Can you create a new set of points that is not a function of the RNG and update the figures? They should match both in Julia v1.6 and v1.7. Thank you for the help 💯

Reply:
Thank you for the review! I've changed it that way.